config/locales/en.yml

```diff
@@ -8,3 +8,4 @@ en:
       processing: "Processing"
       succeeded: "Succeeded"
       failed: "Failed"
+    to_home_page_link: "Back to home page"
```

Comment:
This might be used on more thank one page so moving to shared section could be good:

```yaml
en:
  keywords:
    #...
  shared:
    link:
      back_to_home: "Back to home page"
```

Reply:
Agree. Fixed in d483681c6a18bb3da74436a75d30e434f37cf319